registry.json

```diff
@@ -3656,5 +3656,19 @@
     "telegram": "hungbv2",
     "operatorName": "Bui Viet Hung",
     "github": "sirhung1993"
-  }
-  ]
+  },
+  {
+    "ticker": "ISLA",
+    "poolId": "679b1d8a164791f21ab5a9a7fd2c8cfc108f76aad351912ec841847e",
+    "social": {
+      "website": "https://0xisla.live",
+      "twitter": "IslaStakePool",
+      "facebook": "",
+      "instagram": "",
+      "youtube": ""
+    },
+    "telegram": "islastakepool",
```

Comment:
Please can you change this to the TG handle of an individual and not a group or channel?

Reply:
done! Can you confirm that it's ok? Thank you